theme/assets/js/global.js

```diff
@@ -511,6 +513,7 @@ document.addEventListener('alpine:init', () => {
         locale: Constants.DEFAULT_LOCALE,
         currency: Constants.DEFAULT_CURRENCY,
         defaultFulfillment: Constants.FULFILLMENT_SHIPPING,
+        applicationId: null,
```

Comment:
We also define `applicationId` in the above changes but that one is an empty string, is there any significance to either an empty string vs null?